public/docs/_examples/displaying-data/dart/web/main.dart

```diff
@@ -0,0 +1,16 @@
+// #docregion
+import 'package:angular2/bootstrap.dart';
```

Comment:
Nice. I just did some similar multiple-version work on the Forms example, but I just created copies of the HTML file and didn't make it easy to switch between them. I like this v1/v2/v3/v4 approach. I suppose the only downside is when, even when the HTML is the only thing that changes, you'd have to create extra Dart files to go along with them.